clients/src/test/java/org/apache/kafka/common/utils/ByteUtilsTest.java

```diff
@@ -241,6 +244,119 @@ public void testDouble() throws IOException {
         assertDoubleSerde(Double.NEGATIVE_INFINITY, 0xFFF0000000000000L);
     }
 
+    @Test
+    public void testCorrectnessWriteUnsignedVarlong() {
```

Comment:
Should we remove or disable this test?

Reply:
I would be inclined to keep this test to catch any potential bugs in the future changes to the `WriteUnsignedVarlong` implementation. This test increases the testing coverage over a larger range of values than what the existing tests check for.

Reply:
I agree. So maybe we can `disable` them for now, and add comment for future use like what we did [here](https://github.com/apache/kafka/blob/trunk/core/src/test/scala/unit/kafka/network/SocketServerTest.scala#L1410). After all, the current varLong test is just running the same logic and verify they have the same results, right?

Reply:
Done in the latest commit.